gitfile/tokenfetcer_env_var_test.go

```diff
@@ -0,0 +1,15 @@
+package gitfile
+
+import (
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+func TestCreateHeaderStructFromEnv(t *testing.T) {
+	//t.Setenv("TOKEN", "abcd_foo") // only since 1.17
+	os.Setenv("TOKEN", "abcd_foo")
+	defer os.Unsetenv("TOKEN")
```

Comment:
If the tests are run in parallel, this might mess with other tests relying on `TOKEN` being set.

Reply:
what's the better solution? Unfortunately, `t.Setenv` does not work with 1.16 we're using as a default.